database.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE family;
+
+USE family;
+
+CREATE TABLE `users` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`name` VARCHAR(25) NOT NULL,
+	`status` VARCHAR(50) NOT NULL,
```

Comment:
We agreed to use only 3 possible values for a status 
0,1,2 
do we really need here varchar length 50?